dev-java/velocity/velocity-2.4.1.ebuild

```
@@ -99,6 +99,7 @@ src_prepare() {
 }
 
 src_test() {
+	export TZ="Europe/Berlin"	# https://bugs.gentoo.org/952126
```

```suggestion
	local -x TZ="Europe/Berlin"	# https://bugs.gentoo.org/952126
```
Let's try it this way, it's a bit cleaner.